packages/tools/client-debugger/client-debugger-view/src/RootView.tsx

```diff
@@ -5,6 +5,7 @@
 import React from "react";
 
 import { IMessageRelay } from "@fluid-tools/client-debugger";
+import { FluentProvider, teamsLightTheme } from "@fluentui/react-components";
```

Comment:
We shouldn't have to import specific teamsLightTheme or other theme. 

We have the theme helper: https://github.com/microsoft/FluidFramework/blob/bdbc5c3ff450fab216b283dd4a1839e781334a49/packages/tools/client-debugger/client-debugger-view/src/ThemeHelper.ts#L11 which should be applied globally.

Reply:
I might have missed applying to the root and instead applied to the `Debugger.tsx` file which we need to fix. Just moving the line from https://github.com/microsoft/FluidFramework/blob/bdbc5c3ff450fab216b283dd4a1839e781334a49/packages/tools/client-debugger/client-debugger-view/src/Debugger.tsx#L195 to here should fix the issue and apply theme automatically.

Reply:
I think it shouldn't be necessary here in general; we'd just be applying it "around" the `FluidClientDebuggers` component, which is the one that already wraps all its contents with the theme provider.

That said, we could put it here instead and remove it over there, no strong opinion between those two.

Reply:
Moved the getFluentUIThemeToUse to the RootView